[2024-02-28T15:40:17.033+0530] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_weather_data_1.fetch_files manual__2024-02-28T10:09:59.933071+00:00 [queued]>
[2024-02-28T15:40:17.035+0530] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_weather_data_1.fetch_files manual__2024-02-28T10:09:59.933071+00:00 [queued]>
[2024-02-28T15:40:17.035+0530] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-02-28T15:40:17.039+0530] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): fetch_files> on 2024-02-28 10:09:59.933071+00:00
[2024-02-28T15:40:17.042+0530] {standard_task_runner.py:60} INFO - Started process 5628 to run task
[2024-02-28T15:40:17.044+0530] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'fetch_weather_data_1', 'fetch_files', 'manual__2024-02-28T10:09:59.933071+00:00', '--job-id', '88', '--raw', '--subdir', 'DAGS_FOLDER/Data_fetcher.py', '--cfg-path', '/var/folders/th/tnjyp6yd1cj78nysn5t4jl6h0000gn/T/tmpkao26vrk']
[2024-02-28T15:40:17.044+0530] {standard_task_runner.py:88} INFO - Job 88: Subtask fetch_files
[2024-02-28T15:40:17.056+0530] {task_command.py:423} INFO - Running <TaskInstance: fetch_weather_data_1.fetch_files manual__2024-02-28T10:09:59.933071+00:00 [running]> on host aniks-macbook-air.local
[2024-02-28T15:40:17.072+0530] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='fetch_weather_data_1' AIRFLOW_CTX_TASK_ID='fetch_files' AIRFLOW_CTX_EXECUTION_DATE='2024-02-28T10:09:59.933071+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-28T10:09:59.933071+00:00'
[2024-02-28T15:40:46.808+0530] {python.py:202} INFO - Done. Returned value was: None
[2024-02-28T15:40:46.824+0530] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=fetch_weather_data_1, task_id=fetch_files, execution_date=20240228T100959, start_date=20240228T101017, end_date=20240228T101046
[2024-02-28T15:40:46.862+0530] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-28T15:40:46.875+0530] {taskinstance.py:3309} INFO - 1 downstream tasks scheduled from follow-on schedule check
